src/components/Shop/Navbar.tsx

```diff
@@ -12,9 +20,22 @@ const Navbar: FC = () => {
         />
       </div>
       <div>
-        <span>Ola Nordmann | (icon) xxxkr - xxx Ølcoins (icon)</span>
+        <IconContext.Provider value={{ color: "white", size: "20px" }}>
+          {user ? (
```

Comment:
You can assert that it is not undefined. It is used when you know a value cannot be null/undefined, like this situation. `user!.first_name` & `user!.balance`